quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HttpPropertyMappers.java

```diff
@@ -68,6 +68,7 @@ private HttpPropertyMappers(){}
                         .to("quarkus.http.ssl.protocols")
                         .description("The list of protocols to explicitly enable.")
                         .paramLabel("protocols")
+                        .defaultValue("TLSv1.3")
```

Comment:
Should we also add expectedValues here with TLSv1.2 and TLSv1.3? Not sure if it works, as this would be multi-valued (expectedValues would not be either one or the other, but can include both in different orders). Apart from that, lgtm

Reply:
I wouldn't explicitly recommend though as discussed before?

Reply:
works for now for me. as said, just a question, PR is approved :)